README.md

```diff
@@ -54,14 +54,14 @@ Event listeners are tiggered by these calls:
 - `process.stderr.write(string)`
 - `process.exit(code)`
 
-The console functions `log`, `info`, `warn` and `error` get replaced and the
+The console functions `log` and `error` get replaced and the
 original implementation is exposed:
 
 - `console.log.original`
-- `console.info.original`
-- `console.warn.original`
```

Comment:
Sorry, that was a misunderstanding then. I thought you wanted to fix that console.log / console.info pointed to the same thing...